examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldServerTls.java

```diff
@@ -110,11 +89,16 @@ public static void main(String[] args) throws IOException, InterruptedException
             System.exit(0);
         }
 
+        // If only providing a private key, you can use TlsServerCredentials.create() instead of
+        // interacting with the Builder.
+        TlsServerCredentials.Builder tlsBuilder = TlsServerCredentials.newBuilder()
+            .keyManager(new File(args[1]), new File(args[2]));
+        if (args.length == 4) {
+            tlsBuilder.trustManager(new File(args[3]));
+            tlsBuilder.clientAuth(TlsServerCredentials.ClientAuth.REQUIRE);
```

Comment:
Similar comment as before. How about making it idiomatic by assigning back to `tlsBuilder`?